pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_agent.py

```diff
@@ -154,15 +171,27 @@ async def streamed_response():
         )
         activities.append(self.event_stream_handler_activity)
 
+        if additional_models:
```

Comment:
Let's have all the `_registered_model_ids` stuff in this `__init__` method together, so probably move this up